lib/msal-common/src/utils/Constants.ts

```diff
@@ -64,6 +64,17 @@ export const Constants = {
     INVALID_INSTANCE: "invalid_instance",
 };
 
+export const HttpStatus = {
```

Comment:
Unrelated to this PR, don't we have similar constants in `msal-node`? can we leverage these from common?

Reply:
They were in msal-node. I moved them to msal-common.